app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/MainActivity.kt

```diff
@@ -194,23 +198,23 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun setupTutorialObserver() {
-        tutorialViewModel.showStoresTutorial.observe(this) { isTutorialTriggered ->
-            if (isTutorialTriggered == true && tutorialViewModel.shouldShowStoresTutorial()) {
-                showStoresTutorial()
+        tutorialViewModel.showHistoryTutorial.observe(this) { isTutorialTriggered ->
+            if (isTutorialTriggered == true && tutorialViewModel.shouldShowHistoryTutorial()) {
+                showHistoryTutorial()
             }
         }
     }
 
-    private fun showStoresTutorial() {
-        val storeItem = findViewById<BottomNavigationItemView>(R.id.navigation_stores_graph)
+    private fun showHistoryTutorial() {
+        val storeItem = findViewById<BottomNavigationItemView>(R.id.navigation_history_graph)
```

Comment:
I think we could rename this value as well, either as something generic (bottomNavItem) or something like historyItem

Reply:
good catch, historyItem makes sense!